float32 r_fk_pX
float32 r_fk_pY
float32 r_fk_pZ
float32 l_fk_pX
float32 l_fk_pY
float32 l_fk_pZ
